src/struct_info.rs

```diff
@@ -126,6 +126,13 @@ impl<'a> StructInfo<'a> {
             quote!(#[doc(hidden)])
         };
 
+        let mut b_decl_generics = b_generics.clone();
```

Comment:
Why is this required? `b_generics` is only used when declaring the builder type (and you want the default there) and when splitting it for the `impl` block (where defaults should not be looked at anyway). So why clone it instead of modifying the original?

Reply:
That's redundant, indeed.